plugins/woocommerce/client/blocks/tests/e2e/tests/add-to-cart-with-options/add-to-cart-with-options.block_theme.spec.ts

```diff
@@ -128,18 +128,9 @@ test.describe( 'Add to Cart + Options Block', () => {
 
 		await page.goto( '/hoodie' );
 
-		const logoNoOption = page.getByRole( 'radio', {
-			name: 'No',
-			exact: true,
-		} );
-		const colorBlueOption = page.getByRole( 'radio', {
-			name: 'Blue',
-			exact: true,
-		} );
-		const colorGreenOption = page.getByRole( 'radio', {
-			name: 'Green',
-			exact: true,
-		} );
+		const logoNoOption = page.locator( 'label:has-text("No")' );
+		const colorBlueOption = page.locator( 'label:has-text("Blue")' );
+		const colorGreenOption = page.locator( 'label:has-text("Green")' );
```

Comment:
Is there anything preventing us from using [`getByLabel`](https://playwright.dev/docs/api/class-page#page-get-by-label)?

```suggestion
		const logoNoOption = page.getByLabel( 'No' );
		const colorBlueOption = page.getByLabel( 'Blue' );
		const colorGreenOption = page.getByLabel( 'Green' );
```

Reply:
Yes, `getByLabel()` selects the `<input>`, but then it can't be clicked it because it's visually hidden. I added an explanatory comment in 97fcfb0354fcb00cbbeeff436a093dddb2801a53.

Reply:
Ah, thanks for clarifying! 🙌